adapters/socketio_berkeley.c

```diff
@@ -93,7 +93,7 @@ typedef struct SOCKET_IO_INSTANCE_TAG
     char* target_mac_address;
     IO_STATE io_state;
     SINGLYLINKEDLIST_HANDLE pending_io_list;
-    unsigned char recv_bytes[RECEIVE_BYTES_VALUE];
+    unsigned char recv_bytes[XIO_RECEIVE_BYTES_VALUE];
```

Comment:
Please forgive me :)
Sharing another option for naming: XIO_RECEIVE_BUFFER_SIZE
This would be P237 in the priority scale.